easybuild/easyconfigs/t/TRF/TRF-4.09-linux64.eb

```diff
@@ -0,0 +1,23 @@
+easyblock = 'Binary'
+
+name = 'TRF'
+version = '4.09'
+versionsuffix = '-linux64'
+
+homepage = 'https://tandem.bu.edu/trf/trf.html'
+description = """Tandem repeats finder: a program to analyze DNA sequences. Legacy version."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+#source_urls = ['https://tandem.bu.edu/trf/trf409.linux64.download.html']
+sources = ['trf409.legacylinux64']
+checksums = [('md5', 'f97b6d1fc06adbd1484b384d5481916a')]
+
+postinstallcmds = ["mv %(installdir)s/trf409.legacylinux64 %(installdir)s/trf"]
```

Comment:
@RvDijk please try to avoiding hardcoding the version; replace `409` with `%(version_major)s%(version_minor)s`